source/wp-content/themes/wporg-developer-2023/src/code-changelog/edit.js

```diff
@@ -0,0 +1,10 @@
+import { useBlockProps } from '@wordpress/block-editor';
+import ServerSideRender from '@wordpress/server-side-render';
+
+export default function Edit( { name, attributes } ) {
+	return (
+		<p { ...useBlockProps() }>
```

Comment:
Technically this adds the block wrappers twice, since the ServerSideRender adds them too. You can just return the `ServerSideRender` component.